napari_aicssegmentation/_function.py

```diff
@@ -47,3 +50,7 @@ def image_arithmetic(
 ) -> "LayerDataTuple":
     """Adds, subtracts, multiplies, or divides two same-shaped image layers."""
     return (operation.value(layerA, layerB), {"colormap": "turbo"})
+
+
+def test_sanity():
+    assert True
```

Comment:
Never seen something like this before, how/when do you use this?

Reply:
This is essentially the most trivial unit test you can write. The reason it was added is that if you run tox and you don't have any tests you get a cryptic error message, I think it's Error code 5. 
Did that answer your question or is it a more general question about unit tests?

Reply:
Nope that's good thanks!